mlora/model_llama.py

```diff
@@ -340,22 +360,23 @@ def from_pretrained(path: str,
         llama_args.pad_token_id_ = llama_model.config.pad_token_id
         if llama_args.pad_token_id_ is None:
             llama_args.pad_token_id_ = -1
+        llama_args.dtype = llama_model.dtype
 
         llama_args.device = device
 
         model = LlamaModel(llama_args)
 
         embedding_weight = llama_model.model.embed_tokens.weight.to(
-            device=device).requires_grad_(False)
+            device=device).detach()
```

Comment:
why use detach to copy tensor??
can add some comment ??

Reply:
<img width="748" alt="image" src="https://github.com/TUDB-Labs/multi-lora-fine-tune/assets/16332179/ce433ec6-fb3b-474b-8d36-fadfb5065fbd">